src/language-js/parser-babel.js

```diff
@@ -44,16 +43,41 @@ function babelOptions(extraPlugins = []) {
   };
 }
 
+function resolvePluginsConflict(
+  condition,
+  pluginCombinations,
+  conflictPlugins
+) {
+  const combinations = [...pluginCombinations];
+  if (condition) {
+    for (const combination of pluginCombinations) {
+      for (const plugin of conflictPlugins) {
+        combinations.push([...combination, plugin]);
+      }
+    }
+  }
+  return combinations;
```

Comment:
Let's say the input combinations are `[["jsx", "typescript"], ["typescript"]]`, in that case the output will be:
```js
[
  ["jsx", "typescript"],
  ["typescript"],
  ["jsx", "typescript", ["pipelineOperator", { proposal: "smart" }]],
  ...
]
```
Why do we need the first two elements in this list?